engines/docker/generated_configschema.go

```diff
@@ -0,0 +1,47 @@
+// This source code file is AUTO-GENERATED by github.com/taskcluster/jsonschema2go
+
+package docker
+
+import "github.com/taskcluster/taskcluster-worker/runtime"
+
+type (
+	// Config applicable to docker engine
+	Config struct {
+
+		// Root Volume blah blah
+		RootVolume string `json:"rootVolume"`
+	}
+)
+
+func ConfigSchema() runtime.CompositeSchema {
```

Comment:
I was tempted to do:
`var configSchema = func() runtime.CompositeSchema { .... return ... ; }()`

So it's just a variable...


Also quick question, is it possible to make these types private... I don't want to export them from the package.
Obviously `RootVolume` need to be exported for json to work... but `Config struct {` does not.
Similar for `ConfigSchema()` no need to export it... Also I suspect it's easier and safer to do it as a variable..
That way if there is a panic() inside the function it'll always happen when the program starts, so tests would never work, if the is a panic()...